### ISSUE TITLE
Better Library Exporter with IGDB data

### ISSUE THREAD

**Issue by Michael-Z-Freeman** [JosefNemec/Playnite#1798]
It's great that I can export to CSV but it would be nice to have more exported especially the IGDB info like game description, picture and so forth. This could then be used to make a HTML page of the games from the CSV. I have looked for another solution for doing this using just the exported list of games but I can't find any other tool apart from setting up plugins on Wordpress or Drupal that consult the IGDB API. That then also needs CSV import. Exporting from PlayNite is probably simpler.

**Comment by JosefNemec** [comment]
There were several requests to add some export enhanced export feature, but it's not realistically possible. Everybody has different requirements about data format and we can't cater to all those requirements. That's what extensions system is there for. If you need to export data from Playnite, then you can write an extension that will export it in exact way you want.

**Comment by Michael-Z-Freeman** [comment]
> There were several requests to add some export enhanced export feature, but it's not realistically possible. Everybody has different requirements about data format and we can't cater to all those requirements. That's what extensions system is there for. If you need to export data from Playnite, then you can write an extension that will export it in exact way you want.

OK, thanks. I can't myself write an extension at the moment. The export format seems to use a pretty generic type anyway, "CSV". I don't understand why you can't dump more of what you are already exporting and let the end user worry about what to do with it when importing the CSV elsewhere or processing it in some way. I was just thinking of using a BASH script to process each element into a table and an img tag for the picture. But maybe I can drum up some enthusiasm for some to write an HTML exporter if it comes to it.

**Comment by JosefNemec** [comment]
You can modify existing export script if you need more data exported.

**Comment by Michael-Z-Freeman** [comment]
> You can modify existing export script if you need more data exported.

OK, yes, thanks. I did not fully appreciate that the plugin is simply a script.

I found the rest of the field names in "Playnite-master/doc/api/.manifest" so it looks like a case of simply adding those to a clone of the export plugin although I have not tested this yet. If that works then maybe I, or someone,  might be able to contribute an "Export to web page" plugin one day !
